DEPS

```diff
@@ -21,6 +21,10 @@ deps = {
   "third_party/bitcoin-core/src": "https://github.com/bitcoin/bitcoin.git@95ea54ba089610019a74c1176a2c7c0dba144b1c",
   "third_party/argon2/src": "https://github.com/P-H-C/phc-winner-argon2.git@62358ba2123abd17fccf2a108a301d4b52c01a7c",
   "third_party/rapidjson/src": "https://github.com/Tencent/rapidjson.git@06d58b9e848c650114556a23294d0b6440078c61",
+  "components/playlist/browser/resources/media_detector": {
+    "url": "https://github.com/brave/playlist-component.git@5434730bf7342f1ba5c057f1640882bb38604a85",
+    "condition": "checkout_win or checkout_linux or checkout_mac or checkout_android",
+  }
```

Comment:
let's sort the whole list? and keep comma at the end.

Reply:
Good call!

Reply:
I don't think we should have this list at all, it's problematic as I mentioned here https://github.com/brave/brave-core/pull/16901#discussion_r1152499785